Source/DafnyPipeline.Test/Trivia.cs

```diff
@@ -36,6 +36,7 @@ enum Newlines { LF, CR, CRLF };
 // Trait docstring
 { }
 // This is attached to trait2
+// This is also attached to trait2
```

Comment:
Should we add assertions for these changes?

Reply:
Not necessary, as the comment afterwards is attached to the next token. I was just clarifying the comment :-)